spring-cloud-gcp-starters/spring-cloud-gcp-starter-config/src/main/java/org/springframework/cloud/gcp/config/GoogleConfigPropertySourceLocator.java

```diff
@@ -145,7 +144,6 @@ GoogleConfigEnvironment getRemoteEnvironment() throws IOException, HttpClientErr
 		return new MapPropertySource(PROPERTY_SOURCE_NAME, config);
 	}
 
-	@VisibleForTesting
 	String getProjectId() {
```

Comment:
That's minor and we can live with that, but don't think that I have missed that this method isn't `public` 😉

Reply:
Good catch. PTAL